lib/src/Checkbox/Checkbox.tsx

```diff
@@ -9,19 +9,31 @@ export type CheckboxProps = JSX.IntrinsicElements["input"] & {
     inline?: boolean;
     /** Div wrapper props */
     wrapperProps?: JSX.IntrinsicElements["div"];
+    /** Indeterminate state */
+    indeterminate?: boolean;
     /** Indicator for error, warning or success */
     indicator?: Indicator;
 };
 
-export const Checkbox: React.FC<CheckboxProps> = ({ inline, wrapperProps, indicator, children, ...props }: CheckboxProps) => {
+export const Checkbox: React.FC<CheckboxProps> = ({ inline, wrapperProps, indicator, indeterminate, children, ...props }: CheckboxProps) => {
     const [id, setId] = React.useState<string>(props.id);
 
     React.useEffect(() => setId(props.id || (children ? props.id || randomId("checkbox-") : null)), [props.id, children]);
 
     return (
         <div {...wrapperProps} className={classnames("rc", "checkbox", { inline }, wrapperProps?.className)}>
             <div className={classnames("custom-control", "custom-checkbox", { "custom-control-inline": inline }, { [`is-${indicator?.type}`]: indicator })}>
-                <input {...props} type="checkbox" id={id} className={classnames("custom-control-input", props.className)} />
+                <input
+                    {...props}
+                    type="checkbox"
+                    id={id}
+                    className={classnames("custom-control-input", props.className)}
+                    ref={(input: HTMLInputElement) => {
+                        if (input) {
+                            input.indeterminate = indeterminate;
+                        }
+                    }}
```

Comment:
You need to make sure that `ref` is also passed down to `props.ref`. Add a unit test that verifies that passing both `indeterminate` and `ref` won't affect each other

Reply:
i think we need ForwardedRef for props.ref

Reply:
can you do that please

Reply:
remove indeterminate, and added forwardRef